fjord/feedback/static/js/android_feedback.js

```diff
@@ -0,0 +1,66 @@
+(function($) {
+  'use strict';
+
+  // settings
+  var browserReviewUrl = 'https://play.google.com/store/apps/details?id=org.mozilla.CHANNEL#details-reviews';
+  var deviceReviewUrl = 'market://details?id=org.mozilla.CHANNEL';
+
+  // initialize
+  var reviewUrl, lastUrl = '';
+  var url = decodeURIComponent(window.location.href);
+
+
+  /*
+    Decide whether they're on Fx Android or not
+    (Fx Android's feedback prompt includes a utm param)
+  */
+  var fromDevice = /source=feedback-prompt/.test(url);
+  if (fromDevice) {
+    reviewUrl = deviceReviewUrl;
+    /* 
+      TODO: When bug 1237388 is resolved, remove #play-store here.
+      If url params are as assumed below, it'll Just Work(tm)!
+    */
+    $('#maybe-later, #no-thanks, #play-store').click(
+      function(e) {
+        e.preventDefault();
+        var feedbackEvent = document.createEvent('Events');
```

Comment:
Good call. I tested using a custom nightly mleibovic made for me, plus the console command in https://bugzilla.mozilla.org/show_bug.cgi?id=1235902#c8, and the event-constructed event works great.